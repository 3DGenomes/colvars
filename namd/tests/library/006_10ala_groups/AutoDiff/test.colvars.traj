# step         rmsdA                 ft_rmsdA               rmsdB                 ft_rmsdB              
           0    4.84870372257743e-01  0.00000000000000e+00   4.84870372257743e-01  0.00000000000000e+00 
           1    4.84397558390947e-01  2.73670378085089e+01   4.84397558390947e-01  2.73670378085089e+01 
           2    4.83907428259583e-01  2.13636230062771e+01   4.83907428259583e-01  2.13636230062771e+01 
           3    4.83522392476985e-01  1.44277657519514e+01   4.83522392476985e-01  1.44277657519514e+01 
           4    4.83083112856857e-01  9.02639845600913e+00   4.83083112856857e-01  9.02639845600913e+00 
           5    4.82621016610959e-01  6.12687169260865e+00   4.82621016610959e-01  6.12687169260865e+00 
           6    4.82181848616375e-01  2.94643810676445e+00   4.82181848616375e-01  2.94643810676445e+00 
           7    4.81645520125956e-01  3.27729181515080e+00   4.81645520125956e-01  3.27729181515080e+00 
           8    4.81136761645330e-01  6.92780751596672e+00   4.81136761645330e-01  6.92780751596672e+00 
           9    4.80598465150393e-01  1.27654845240536e+01   4.80598465150393e-01  1.27654845240536e+01 
          10    4.80094226695461e-01  2.25532360920921e+01   4.80094226695461e-01  2.25532360920921e+01 
          11    4.79704703208599e-01  3.31753145481933e+01   4.79704703208599e-01  3.31753145481933e+01 
          12    4.79506792183813e-01  4.03757393152715e+01   4.79506792183813e-01  4.03757393152715e+01 
          13    4.79327275850325e-01  4.51104576283605e+01   4.79327275850325e-01  4.51104576283605e+01 
          14    4.79314968874545e-01  5.00135156793850e+01   4.79314968874545e-01  5.00135156793850e+01 
          15    4.79397379647665e-01  4.89320025041134e+01   4.79397379647665e-01  4.89320025041134e+01 
          16    4.79394129016958e-01  4.68394111879974e+01   4.79394129016958e-01  4.68394111879974e+01 
          17    4.79404085661514e-01  4.28905605743649e+01   4.79404085661514e-01  4.28905605743649e+01 
          18    4.79306164127760e-01  3.83747642964369e+01   4.79306164127760e-01  3.83747642964369e+01 
          19    4.79258291379495e-01  3.45363979139421e+01   4.79258291379495e-01  3.45363979139421e+01 
          20    4.79299937828241e-01  2.86559921546366e+01   4.79299937828241e-01  2.86559921546366e+01 
