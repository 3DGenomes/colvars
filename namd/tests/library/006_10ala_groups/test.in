colvarsTrajFrequency    1

indexFile ../Common/da.ndx


colvar {
    name rmsdA

    outputSystemForce on
    #  outputAppliedForce on
    width 1.0 

    lowerboundary 0.0
    upperboundary 5.0

    lowerwallconstant 100.0
    upperwallconstant 100.0

    rmsd {
        atoms {
            indexGroup Protein_C-alpha
        }
        refPositionsFile ../Common/da.xyz
    }
} 


colvar {
    name rmsdB

    outputSystemForce on
    #  outputAppliedForce on
    width 1.0 

    lowerboundary 0.0
    upperboundary 5.0

    lowerwallconstant 100.0
    upperwallconstant 100.0

    rmsd {
        atoms {
            indexGroup Protein_C-alpha
        }
        refpositionsfile ../Common/da.xyz
    }
} 


harmonic {
    colvars        rmsdA rmsdB
    forceConstant  1.0
    centers        0.0   0.0
}
