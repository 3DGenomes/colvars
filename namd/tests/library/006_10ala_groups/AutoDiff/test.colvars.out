colvars: ----------------------------------------------------------------------
colvars: Please cite Fiorin et al, Mol Phys 2013:
colvars:  http://dx.doi.org/10.1080/00268976.2013.813594
colvars: in any publication based on this calculation.
colvars: SMP parallelism is available.
colvars: ----------------------------------------------------------------------
colvars: Reading new configuration from file "test.in":
colvars: # indexFile = "../Common/da.ndx"
colvars: The following index groups were read from the index file "../Common/da.ndx":
colvars:   Protein (104 atoms).
colvars:   Protein_noH (51 atoms).
colvars:   Protein_Backbone (40 atoms).
colvars:   Protein_C-alpha (10 atoms).
colvars:   Protein_C-alpha_1_2 (2 atoms).
colvars:   Protein_C-alpha_9_10 (2 atoms).
colvars:   Protein_C-alpha_1 (1 atoms).
colvars:   Protein_C-alpha_2 (1 atoms).
colvars:   Protein_C-alpha_3 (1 atoms).
colvars:   Protein_C-alpha_4 (1 atoms).
colvars:   Protein_C-alpha_5 (1 atoms).
colvars:   Protein_C-alpha_6 (1 atoms).
colvars:   Protein_C-alpha_7 (1 atoms).
colvars:   Protein_C-alpha_8 (1 atoms).
colvars:   Protein_C-alpha_9 (1 atoms).
colvars:   Protein_C-alpha_10 (1 atoms).
colvars: # analysis = off [default]
colvars: # colvarsTrajFrequency = 1
colvars: # colvarsRestartFrequency = 0 [default]
colvars: # colvarsTrajAppend = off [default]
colvars: # scriptedColvarForces = off [default]
colvars: # scriptingAfterBiases = on [default]
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new collective variable.
colvars:   # name = "rmsdA"
colvars:   Initializing a new "rmsd" component.
colvars:     # componentCoeff = 1 [default]
colvars:     # componentExp = 1 [default]
colvars:     # period = 0 [default]
colvars:     # wrapAround = 0 [default]
colvars:     # scalable = on [default]
colvars:     Scalable calculation is not available for group "atoms" with the current configuration.
colvars:       Initializing atom group "atoms".
colvars:       # indexGroup = "Protein_C-alpha"
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableForces = on [default]
colvars:       # centerReference = off [default]
colvars:       # rotateReference = off [default]
colvars:       # enableFitGradients = on [default]
colvars:       Atom group "atoms" defined, 10 atoms initialized: total mass = 120.11, total charge = 0.53.
colvars:     # refPositions =  [default]
colvars:     # refPositionsFile = "../Common/da.xyz"
colvars:     # refPositionsCol = "" [default]
colvars:     Enabling "centerReference" and "rotateReference", to minimize RMSD before calculating it as a variable: if this is not the desired behavior, disable them explicitly within the "atoms" block.
colvars:     This is a standard minimum RMSD, derivatives of the optimal rotation will not be computed as they cancel out in the gradients.
colvars:   All components initialized.
colvars:   # width = 1
colvars:   # lowerBoundary = 0
colvars:   # lowerWallConstant = 100
colvars:   # lowerWall = 0 [default]
colvars:   # upperBoundary = 5
colvars:   # upperWallConstant = 100
colvars:   # upperWall = 5 [default]
colvars:   # hardLowerBoundary = off [default]
colvars:   # hardUpperBoundary = off [default]
colvars:   # expandBoundaries = off [default]
colvars:   # timeStepFactor = 1 [default]
colvars:   # extendedLagrangian = off [default]
colvars:   # outputValue = on [default]
colvars:   # outputVelocity = off [default]
colvars:   # outputSystemForce = on
colvars:   # outputAppliedForce = off [default]
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new collective variable.
colvars:   # name = "rmsdB"
colvars:   Initializing a new "rmsd" component.
colvars:     # componentCoeff = 1 [default]
colvars:     # componentExp = 1 [default]
colvars:     # period = 0 [default]
colvars:     # wrapAround = 0 [default]
colvars:     # scalable = on [default]
colvars:     Scalable calculation is not available for group "atoms" with the current configuration.
colvars:       Initializing atom group "atoms".
colvars:       # indexGroup = "Protein_C-alpha"
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableForces = on [default]
colvars:       # centerReference = off [default]
colvars:       # rotateReference = off [default]
colvars:       # enableFitGradients = on [default]
colvars:       Atom group "atoms" defined, 10 atoms initialized: total mass = 120.11, total charge = 0.53.
colvars:     # refPositions =  [default]
colvars:     # refPositionsFile = "../Common/da.xyz"
colvars:     # refPositionsCol = "" [default]
colvars:     Enabling "centerReference" and "rotateReference", to minimize RMSD before calculating it as a variable: if this is not the desired behavior, disable them explicitly within the "atoms" block.
colvars:     This is a standard minimum RMSD, derivatives of the optimal rotation will not be computed as they cancel out in the gradients.
colvars:   All components initialized.
colvars:   # width = 1
colvars:   # lowerBoundary = 0
colvars:   # lowerWallConstant = 100
colvars:   # lowerWall = 0 [default]
colvars:   # upperBoundary = 5
colvars:   # upperWallConstant = 100
colvars:   # upperWall = 5 [default]
colvars:   # hardLowerBoundary = off [default]
colvars:   # hardUpperBoundary = off [default]
colvars:   # expandBoundaries = off [default]
colvars:   # timeStepFactor = 1 [default]
colvars:   # extendedLagrangian = off [default]
colvars:   # outputValue = on [default]
colvars:   # outputVelocity = off [default]
colvars:   # outputSystemForce = on
colvars:   # outputAppliedForce = off [default]
colvars: ----------------------------------------------------------------------
colvars: Collective variables initialized, 2 in total.
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new "harmonic" instance.
colvars:   # name = "harmonic1" [default]
colvars:   # colvars = { rmsdA, rmsdB }
colvars:   # outputEnergy = off [default]
colvars:   # forceConstant = 1
colvars:   # centers = { 0, 0 }
colvars:   # targetCenters = { 0, 0 } [default]
colvars:   # targetForceConstant = 0 [default]
colvars:   # outputCenters = off [default]
colvars: ----------------------------------------------------------------------
colvars: Collective variables biases initialized, 1 in total.
colvars: ----------------------------------------------------------------------
colvars: Collective variables module (re)initialized.
colvars: ----------------------------------------------------------------------
colvars: Updating NAMD interface:
colvars: updating atomic data (10 atoms).
colvars: updating group data (0 scalable groups, 0 atoms in total).
colvars: Re-initialized atom group rmsdA:0/0. 10 atoms: total mass = 120.11.
colvars: Re-initialized atom group rmsdB:0/0. 10 atoms: total mass = 120.11.
colvars: The final output state file will be "test.colvars.state".
colvars: Writing to colvar trajectory file "test.colvars.traj".
colvars: Updating NAMD interface:
colvars: updating atomic data (10 atoms).
colvars: updating group data (0 scalable groups, 0 atoms in total).
colvars: Saving collective variables state to "test.colvars.state".
